src/components/public/BorderWithShadow.tsx

```diff
@@ -0,0 +1,17 @@
+import { Card } from "@mui/material";
+import { ReactNode } from "react";
+
+
+type props = {
+    children: ReactNode
+}
+export default function BorderWithShadow(props: props) {
+    return <Card sx={{
+        border: "solid 4px",
+        borderRadius: "16px",
+        boxShadow: "8px 8px #BFBFBF",
```

Comment:
I've seen that there are many places that use `#BFBFBF`. May be try to add it to `_app.tsx`?

Reply:
now i add it to _app.tsx